### PR TITLE
feat: increase code coverage for retryHandlerFactory [In Progress]

### PR DESCRIPTION
**Pull request by REETI-SARUP**
### Checklist
🚨 Please review this repository's [contribution guidelines](./CONTRIBUTING.md).

- [ ] I've read and agree to the project's contribution guidelines.
- [ ] I'm requesting to **pull a topic/feature/bugfix branch**.
- [ ] I checked that my code additions will pass code linting checks and unit tests.
- [ ] I updated unit and integration tests (if applicable).
- [ ] I'm ready to notify the team of this contribution.

### Description
**What does this change do and why?**
Increase code coverage by adding tests for RetryHandlerFactory

Fixes https://github.com/intuit/RWebPulse/issues/33

[Link to JIRA]

Thank you!

### CODE REVIEW

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  public void setUp() {
+    retryHandlerFactory = new RetryHandlerFactory();
+  }
+
+  @Test
+  public void testAddAndGetHandler() {
+    RetryHandler retryHandler = new DefaultRetryHandler();
+    String handlerTestName = "default";
+
+    retryHandlerFactory.addHandler(handlerName, retryHandler);
```
Comment (aditi2205) [inline src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java:22]: should be handlerTestName

Comment (REETI-SARUP) [reply src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java]: Yeah, typo, fixed it. 👍

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  private RetryHandlerFactory retryHandlerFactory;
+
+  @BeforeEach
+  public void setUp() {
+    retryHandlerFactory = new RetryHandlerFactory();
+  }
+
+  @Test
+  public void testAddAndGetHandler() {
+    RetryHandler retryHandler = new DefaultRetryHandler();
```
Comment (aditi2205) [inline src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java:19]: DefaultRetryHandler is not imported in this class

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  public void setUp() {
+    retryHandlerFactory = new RetryHandlerFactory();
+  }
+
+  @Test
+  public void testAddAndGetHandler() {
+    RetryHandler retryHandler = new RetryHandler();
+    String handlerTestName = "default";
+
+    retryHandlerFactory.addHandler(handlerTestName, retryHandler);
```
Comment (aditi2205) [inline src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java:22]: Static class, directly call RetryHandlerFactory.addhandler()

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import org.junit.jupiter.api.Test;
+
+
+public class RetryHandlerFactoryTest {
+  
+  private RetryHandlerFactory retryHandlerFactory;
+
+  @BeforeEach
+  public void setUp() {
+    retryHandlerFactory = new RetryHandlerFactory();
```
Comment (aditi2205) [inline src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java:14]: No need, directly use static methods with classname

**Comment by REETI-SARUP** [comment]
Closing this PR, will raise a fresh updated one separately.

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java (hunk 4 above)
Comment (REETI-SARUP) [reply src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java]: Updated

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java (hunk 3 above)
Comment (REETI-SARUP) [reply src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java]: Updated the latest changes

File: src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java (hunk 2 above)
Comment (REETI-SARUP) [reply src/test/java/com/intuit/springwebclient/retryHandler/RetryHandlerFactoryTest.java]: Updated the PR, handled this.

**Pull request closed without merging**